src/pages/admin/pending-publication.tsx

```diff
@@ -18,6 +61,12 @@ export default function MyContributions() {
         authenticatedOnly={[UserRole.ADMIN]}
       >
         <ResourceList queryResult={queryResult} useFilters showEditProposals />
+        <div className="my-8 lg:max-w-sm">
```

Comment:
While this is a good idea in theory, I don't think it's worth the additional complexity in the codebase because the same thing can be easily done directly from prisma studio.

Reply:
Thats fair, I guess I felt a little guilty about not having any way to manage drafts as an admin within the app. Ive removed it